### PR TITLE
Add Readme

### LINKED ISSUE #1: Create a readme file
This is an old project I did for fun to learn web development.
As a first issue I need to add a readme file to this repo.

### PR DESCRIPTION
**Pull request by trybick**
Add Readme file. Here is a preview:

Let me know if you want any changes.

Screen Shot 2022-10-09 at 1 28 36 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Contributors
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```
Comment (eduardconstantin) [inline README.md]: Add an extra line at the end

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Review by eduardconstantin — APPROVED** [review]
Great! Thank you!

File: README.md (hunk 1 above)
Comment (trybick) [reply README.md]: Just added a line there. VSCode shows the line (see screenshot) but github is not showing it 
Screen Shot 2022-10-09 at 1 40 14 PM

**Comment by eduardconstantin** [comment]
@all-contributors please add @trybick for doc

**Pull request merged**